Frontend/ui-driver/src/Services/APITypes.purs

```diff
@@ -1202,3 +1203,218 @@ instance showGetGetPerformanceRes :: Show GetPerformanceRes where show = generic
+    }
+  }
+
+instance makeGetOptionsReq :: RestEndpoint GetOptionsReq GetOptionsRes where
+    makeRequest reqBody@(GetOptionsReq categoryId language) headers = defaultMakeRequest GET (EP.getOptions categoryId language) headers reqBody
+    decodeResponse    = decodeJSON
+    encodeRequest req = defaultEncode req
+
+derive instance genericGetOptionsReq :: Generic GetOptionsReq _
+instance showGetOptionsReq     :: Show GetOptionsReq where show     = genericShow
+instance standardGetOptionsReq :: StandardEncode GetOptionsReq where standardEncode (GetOptionsReq _ _) = standardEncode {}
+instance decodeGetOptionsReq   :: Decode GetOptionsReq where decode = defaultDecode
+instance encodeGetOptionsReq   :: Encode GetOptionsReq where encode = defaultEncode
+
+derive instance genericGetOptionsRes :: Generic GetOptionsRes _
+instance showGetGetOptionsRes        :: Show GetOptionsRes where show     = genericShow
+instance standardEncodeGetOptionsRes :: StandardEncode GetOptionsRes where standardEncode (GetOptionsRes req) = standardEncode req
+instance decodeGetOptionsRes         :: Decode GetOptionsRes where decode = defaultDecode
+instance encodeGetOptionsRes         :: Encode GetOptionsRes where encode = defaultEncode
+
+--------------------------------------------------- IssueReport ----------------------------------------------------
+newtype PostIssueReq = PostIssueReq
+  { optionId :: Maybe String
+  , rideId :: Maybe String
+  , categoryId :: String
+  , mediaFiles :: Array String
+  , description :: String
+  }
+
+newtype PostIssueRes = PostIssueRes { issueReportId :: String }
+
+instance makePostIssueReq :: RestEndpoint PostIssueReq PostIssueRes where
+    makeRequest reqBody@(PostIssueReq issueDetails) headers = defaultMakeRequest POST (EP.postIssue "") headers reqBody
+    decodeResponse    = decodeJSON
+    encodeRequest req = defaultEncode req
+
+derive instance genericPostIssueReq :: Generic PostIssueReq _
+instance showPostIssueReq     :: Show PostIssueReq where show     = genericShow
+instance standardPostIssueReq :: StandardEncode PostIssueReq where standardEncode (PostIssueReq req) = standardEncode {}
+instance decodePostIssueReq   :: Decode PostIssueReq where decode = defaultDecode
+instance encodePostIssueReq   :: Encode PostIssueReq where encode = defaultEncode
+
+derive instance genericPostIssueRes :: Generic PostIssueRes _
+instance showGetPostIssueRes        :: Show PostIssueRes where show     = genericShow
+instance standardEncodePostIssueRes :: StandardEncode PostIssueRes where standardEncode (PostIssueRes res) = standardEncode res
+instance decodePostIssueRes         :: Decode PostIssueRes where decode = defaultDecode
+instance encodePostIssueRes         :: Encode PostIssueRes where encode = defaultEncode
+
+--------------------------------------------------- IssueInfo ----------------------------------------------------
+newtype IssueInfoReq = IssueInfoReq String
+
+newtype IssueInfoRes = IssueInfoRes
```

Comment:
@ThisIsMani @mohdshoaib18169 Can you check response type in swagger and verify if its same

Reply:
I checked in Postman (I am not able to access swagger). There are few more fields, which are not required so I did not use them here.